core/src/main/java/com/sequenceiq/cloudbreak/service/ClusterCommonService.java

```diff
@@ -127,7 +129,14 @@ private FlowIdentifier clusterHostgroupAdjustmentChange(Long stackId, UpdateClus
                     updateJson.getHostGroupAdjustment().getHostGroup(), stack.getName()));
         }
         if (blueprintService.isClouderaManagerTemplate(blueprint)) {
-            cmTemplateValidator.validateHostGroupScalingRequest(blueprint, hostGroup.get(),
+            User creator = stack.getCreator();
+            String userCrn = creator.getUserCrn();
```

Comment:
Shouldn't we gather and check the user who initiated this scaling operation?

Reply:
I think what we want to check is the account entitlement. It does not matter who initiated this

Reply:
Yet, but probably it we should consider it later, I am ok with that, but I said :)